lib/ansible/module_utils/basic.py

```diff
@@ -105,14 +107,6 @@
 # Python2 & 3 way to get NoneType
 NoneType = type(None)
 
-# Note: When getting Sequence from collections, it matches with strings.  If
-# this matters, make sure to check for strings before checking for sequencetype
-try:
-    from collections.abc import KeysView
-    SEQUENCETYPE = (Sequence, frozenset, KeysView)
-except ImportError:
-    SEQUENCETYPE = (Sequence, frozenset)
```

Comment:
We'll need to leave this in that we've reverted the isinstance changes in the rest of the file.

Reply:
Doesn't `import SEQUENCETYPE` thing work as well?